docs/FAQ.md

```diff
@@ -41,4 +63,5 @@ cd deployment/
 * Status: pull access denied for dna-base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied, Code: 1
 * Status: invalid from flag value dna-base:frontend: pull access denied for dna-base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied, Code: 1
 ```
-If you are facing the above issue, it could be reason of timeout of node modules installing inside the docker image. Please run same the docker-compose up command again.
+
+If you are facing the above issue, it could be the reason for the timeout of node modules installed inside the docker image. Please run same the docker-compose up command again.
```

Comment:
Should this be like the following?
```suggestion
If you are facing the above issue, it could be the reason for the timeout of node modules installed inside the docker image. Please run same the docker-compose up command again. Please run the same docker-compose up command again. 
```